smarts/core/remote_agent_buffer.py

```diff
@@ -216,8 +215,8 @@ def spawn_local_zoo_manager(port):
 def get_manager_channel_stub(addr):
     channel = grpc.insecure_channel(f"{addr[0]}:{addr[1]}")
     try:
-        # Wait until the grpc server is ready or timeout after 30 seconds
-        grpc.channel_ready_future(channel).result(timeout=30)
+        # Wait until the grpc server is ready or timeout after 10 seconds
+        grpc.channel_ready_future(channel).result(timeout=10)
```

Comment:
I think the timeout should not be decided at this level. I would suggest requiring it as an argument.